src/rma/fixtures/admin_index.json

```diff
@@ -1,64 +1,62 @@
-/usr/local/lib/python3.8/site-packages/django/core/management/commands/dumpdata.py:153: ProxyModelWarning: admin_index.ContentTypeProxy is a proxy model and won't be serialized.
-  warnings.warn(
-[
-{
-    "model": "admin_index.appgroup",
-    "pk": 3,
-    "fields": {
-        "order": 2,
-        "name": "Authenticatie en autorisatie",
-        "slug": "auth",
-        "models": [
-            15,
-            14,
-            4
-        ]
-    }
-},
-{
-    "model": "admin_index.appgroup",
-    "pk": 5,
-    "fields": {
-        "order": 4,
-        "name": "Vernietiging",
-        "slug": "destruction",
-        "models": [
-            16,
-            20,
-            17,
-            19,
-            18
-        ]
-    }
-},
-{
-    "model": "admin_index.appgroup",
-    "pk": 10,
-    "fields": {
-        "order": 9,
-        "name": "Configuratie",
-        "slug": "configuratie",
-        "models": [
-            6,
-            21,
-            8,
-            13
-        ]
-    }
-},
-{
-    "model": "admin_index.appgroup",
-    "pk": 11,
-    "fields": {
-        "order": 10,
-        "name": "Logs",
-        "slug": "logs",
-        "models": [
-            10,
-            11,
-            22,
-            12
-        ]
-    }
-}
-]
+[
+{
+    "model": "admin_index.appgroup",
+    "pk": 3,
+    "fields": {
+        "order": 2,
+        "name": "Authenticatie en autorisatie",
+        "slug": "auth",
+        "models": [
+            15,
```

Comment:
models for different developers can have different pks
Please, use the new file `bin/generate_admin_index_fixture` for it, it has all the necessary options to replace pks with natural keys